certified-connectors/QPP NG Integration Hub/ReadMe.md

```diff
@@ -1,57 +1,56 @@
-### GetUserByName
-Retrieve user metadata based on the username.
+Below is the list of operations supported by the QPP-NG connector – 
 
-### IsGroupMember
-Check if a user is a member of a specific group based on user ID and group ID.
+### Get User by Name
+This operation retrieves user information based on the username. 
 
-### GetUser
-Fetch metadata for a specified user.
+### Is Group Member
+This operation checks if a user is a member of a specified group based on user ID and group ID.
 
-### GetAsset
-Obtain metadata for a specific asset.
+### Get Group Members
+This operation will return list of users and all their metadata specific to group Id.
 
-### GetCollectionGroups
-Retrieve applicable routing groups for a given collection, workflow, and status.
+### Get Group By Name
+This operation will return a group (group details) specific to group name.
 
-### SetAttributeValues
-Set attribute values for the QPP Platform.
+### Get User
+This operation retrieves detailed information about a specific user. 
 
-|Parameter|Defintion|Example|
-|:----|:----|:----|
-|name|Url for the Host environment that This Action needs to points to.|{HostNameHere}.app.quark.com|
-|assetId|QPP Platform Asset Id.|{Asset Id here}|
-|Attribute Array|An array of attributes or Single attribute|Click on the Switch to detail Input buttom resmbling a file.
-Copy and paste the Below json formatted using online json compiler to the Body param in SetAttributeValues action after removing the comments inside //
-//Below example is to set Routed to Value to an asset
-[
-  {
-    "attributeId": 25,
-    "type": 7,
-    "multiValued": true,
-    "attributeValue": {
-      "domainValues": [
-        {
-          "id": 5015			//Id For the User or Group Id can be given here. Instead of Id "name" property can be added and used
-        }
-      ]
-    }
-  }
-]
-//Other examples are below
-//To set Status for an Asset
+### Get Asset
+This operation fetches information about a specific asset. More information can be found on the [Quark Developer portal](https://qppngsdk-rel3x.app.quark.com/Documentation/platformServices/restapi/api.html#Quark_application_API_3_0_0_SNAPSHOT_getAsset).
+
+### Get Collection Groups
+This operation retrieves the routing groups associated with a collection, workflow, and status. More information on the [Quark Developer portal](https://qppngsdk-rel3x.app.quark.com/Documentation/platformServices/restapi/api.html#Quark_application_API_3_0_0_SNAPSHOT_getCollectionGroups).
+
+
+### Set Attribute Values
+This operation sets the attribute values for a specific entity or object. More information on the [Quark Developer portal](https://qppngsdk-rel3x.app.quark.com/Documentation/platformServices/restapi/api.html#Quark_application_API_3_0_0_SNAPSHOT_setAttributeValues).
+
+| Parameter          | Defintion                                                     | Example                                                         	   |
```

Comment:
Please correct the spelling "Defintion" in this file

Reply:
Done